packages/react/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -46,15 +47,21 @@ const _Item = forwardRef<HTMLAnchorElement, ItemProps>(
   ({'aria-current': ariaCurrent, className, children, href, selected, ...rest}, ref) => {
     return (
       <li className={styles.Breadcrumbs__item}>
-        <InlineLink
-          href={href}
-          ref={ref}
-          className={clsx(styles.Breadcrumbs__link, selected && styles['Breadcrumbs__link--selected'], className)}
-          aria-current={ariaCurrent ? ariaCurrent : selected ? 'page' : undefined}
-          {...rest}
-        >
-          {children}
-        </InlineLink>
+        {selected ? (
+          <Text
+            size="100"
+            variant="muted"
+            className={clsx(styles.Breadcrumbs__link, styles['Breadcrumbs__link--selected'], className)}
+            aria-current={ariaCurrent ? ariaCurrent : 'page'}
```

Comment:
Nit, but you could simplify this to


```suggestion
            aria-current="page"
```

and remove 'aria-current' from the props destructure above.

If the user passes their own `aria-current` to `Breadcrumbs.Item` then that will override this on the line below as part of `{...rest}`